geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -57,9 +58,7 @@
         "If numexpr is not installed and you need it, install it."
     )
 
-DONT_AUTOGEN_GEOLOCATION = False
-if os.getenv("DONT_AUTOGEN_GEOLOCATION"):
-    DONT_AUTOGEN_GEOLOCATION = True
+DONT_AUTOGEN_GEOLOCATION = PATHS["DONT_AUTOGEN_GEOLOCATION"]
```

Comment:
Is this the same behavior as prior? The previous behavior set this to `True` no matter what the value of "DONT_AUTOGEN_GEOLOCATION" was, as long as it was set.